uo_sys_framework/ACREModule/cfg3DEN/ACRE/Opfor.hpp

```diff
@@ -10,7 +10,7 @@ class OPFOR_ACRE_RADIOTYPES_CAT
             displayName = "Short Range Radio Type";
             tooltip = "Sets the short range radio type for the side";
             control = "UO_FW_ACRE_SR_OPFOR_SELECT";
-            expression="missionNamespace setVariable ['%s', _value]; %s = _value;";
+            expression="missionNamespace setVariable ['%s', _value, true]; %s = _value;";
```

Comment:
Must all these variables set to public at mission start? That  is quite a bit of network traffic.

Reply:
Alot of these extra publics come from the issue of synchronizing everything. These seem to not be necessary anymore with setDefaults working properly.

Reply:
> 
> 
> Alot of these extra publics come from the issue of synchronizing everything. These seem to not be necessary anymore with setDefaults working properly.

If we can get a definite result from testing, and setDefaults now works properly according to that testing then we should definitely drop the public status of these variables.

Reply:
I agree, there is no reason these have to be public

Reply:
All entity expressions not local to the server should be replaced with `expression = "if !(is3DEN) then {_propertyName = '%s'; [_this, _propertyName, _value, false] call UO_FW_fnc_setInitVar;};";`

true 4th parameter sets it to global (execnextframe setvariable for all machines) false sets it to a targeted/local to object setvariable call. Much better than setting a bunch of vars to public in the init.